module-3/backend/week_one.md

```diff
@@ -5,22 +5,37 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is `json`, what does it stand for, and why is it important?
+JavaScript Object Notation; used for APIs, both human and machine readable
+
 2. What's the difference between `joins` and `includes` in ActiveRecord?
+Joins brings two tables together; includes only joins a certain column of another table meaning faster queries
```

Comment:
Includes eager loads related data to avoid additional queries when using the returned data.